### PR TITLE
Replace RDFS annotations with SKOS annotations - Fixes #351

### LINKED ISSUE #351: Implement selected set of SKOS annotations in place of rdfs:comment and rdfs:label
Depends on #85.

### PR DESCRIPTION
**Pull request by uscholdm**
Addressed issue https://github.com/semanticarts/gist/issues/351

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -1927,18 +1899,18 @@ gist:PhysicalIdentifiableItem
...
-		;
 	owl:disjointWith
 		gist:SchemaMetaData ,
 		gist:TimeInstant ,
 		gist:UnitOfMeasure
 		;
+	skos:definition "You could, at least in principle, put an RFID tag on members of this class. Physical things are made of something.  E.g., statues are made of bronze."^^xsd:string ;
+	skos:example
+		"Negative example: A discontinuous thing like a manufacturing line cannot reasonably have an RFID attached to it, even though its parts are not the same kind of thing as the whole."^^xsd:string ,
+		"a computer, a book."^^xsd:string
```
Comment (sa-bpelakh) [inline gistCore.ttl:1910]:
```suggestion
		"A computer, a book."^^xsd:string
```

**Review by sa-bpelakh — APPROVED** [review]
@ungricht I reviewed lines 1000-2000 (GeoPoint through ProductCategory), only one minor comment on capitalization. 
@uscholdm There are 7 terms that do not have a `skos:definition`:
- ControllerType
- universalTime
- universalDateTime
- universalDate
- localTime
- localDateTime
- localDate

We should address those as part of this effort.

**Comment by uscholdm** [comment]
> There are 7 terms that do not have a skos:definition:

I was not sure whether to do that now or in the next round. I'll add them now.

**Comment by rjyounes** [comment]
@uscholdm I don't think you should do that now because the reviewers' task is not to evaluate the annotation values themselves, but to make sure they have been appropriately converted to SKOS. This makes it harder to know what to review. I believe we have one or more existing issues for rewriting annotations.

**Comment by uscholdm** [comment]
In those few cases where there was an existing rdfs:comment that turned out to be nothing like a definition and the end results was no skos:definition, I added a few.  I drew the line at making changes to existing definitions. I could have gone either way. Don't think it matters much. I just made the change.

**Review by rjyounes — CHANGES_REQUESTED** [review]
I'm not doing a full review now, but a note needs to be added to the release notes file. Please remember to include the legacy file in the note.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Nice job! A few very small changes requested.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 
-Release 9.4.1
+Release X.x.x
 -----
+### Minor Updates
+*Converted rdfs annotations to skos annotations*
+
+- `rdfs:label` was replaced with `skos:prefLabel`
+- `rdfs:comment` was replaced with one of `skos:definition`, `skos:example`, `skos:scopeNote` or `skos:altLabel`
+- Backward compatibility is retained because we also publish an auto-generated version of the skos annotations in the file: rdfsAnnotations.ttl.  The diffferent skos annotations are distinguished by a suitable prefix to the `rdfs:comment`. One of: 'ALT: ' 'DEFINITION: ' , 'EXAMPLE: ', 'NOTE: '
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply docs/ReleaseNotes.md:11]: Good catch.

File: gistCore.ttl [hunk 3]
```diff
@@ -341,6 +333,16 @@ gist:CoherentUnit
 			gist:CoherentRatioUnit
 		) ;
 	] ;
+	skos:definition "A unit that is expressed in units that have no conversions.  It may be a simple unit.  It may also be a product or ratio unit that bottoms out in simple units."^^xsd:string ;
+	skos:example
+		"a simple unit: kilogram"^^xsd:string ,
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply gistCore.ttl:338]: I just made the few changes right here (TBC), but a broader automated check would be worth doing. Part of style checking to go into the onto-tools?

File: gistCore.ttl [hunk 4]
```diff
@@ -341,6 +333,16 @@ gist:CoherentUnit
...
+		"the standard unit for acceleration is meters per square second (feet per square second  requires a conversion)"^^xsd:string
+		;
+	skos:prefLabel "Coherent Unit"^^xsd:string ;
+	skos:scopeNote
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply gistCore.ttl:342]: Leaving this alone. For classes the prefLabel should be capitalized start, but not necessarily for properties. Solution is to add automated style check to onto-tools.

File: gistCore.ttl [hunk 5]
```diff
@@ -794,12 +782,13 @@ gist:Duration
 			]
 		) ;
 	] ;
+	skos:definition "Time, but not on a timeline."^^xsd:string ;
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply gistCore.ttl:785]: Good catch.

File: gistCore.ttl [hunk 6]
```diff
@@ -957,33 +946,28 @@ gist:FormattedContent
...
 gist:GeneralMediaType
 	a owl:Class ;
 	rdfs:subClassOf gist:Category ;
-	rdfs:label "General Media Type"^^xsd:string ;
-	rdfs:comment
-		"EXAMPLE: audio, still image, video, textual, physical (e.g., a statue), or performance (i.e. a play).  Or it could be oil or pastel for a painting."^^xsd:string ,
-		"The real-world media type for content."^^xsd:string
-		;
+	skos:definition "The real-world media type for content."^^xsd:string ;
+	skos:example "audio, still image, video, textual, physical (e.g., a statue), or performance (i.e. a play).  Or it could be oil or pastel for a painting."^^xsd:string ;
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply gistCore.ttl:964]: Fixed

File: gistCore.ttl [hunk 7]
```diff
@@ -3490,218 +3436,222 @@ gist:hasUniqueSuperCategory
...
+	skos:prefLabel "Longitude"^^xsd:string ;
 	.
 
 gist:madeUpOf
 	a owl:ObjectProperty ;
-	rdfs:label "Made Up Of"^^xsd:string ;
-	rdfs:comment "as in the vase is made up of clay"^^xsd:string ;
 	rdfs:range gist:PhysicalSubstance ;
+	skos:definition "Relates something to the the substance that it is made up of."^^xsd:string ;
+	skos:example "the vase is made up of clay"^^xsd:string ;
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply gistCore.ttl:3529]: Fixed

File: gistCore.ttl (hunk 4 above)
[the remark this answers is not in the document]
Comment (sa-bpelakh) [reply gistCore.ttl:342]: We can REGEX for capitalization either through SHACL or through query. If you want to write this up as an issue, we can address it between onto_tool and the bundle script.

File: gistCore.ttl [hunk 8]
```diff
@@ -3860,81 +3805,86 @@ gist:toAgent
 			gist:Person
 		) ;
 	] ;
+	skos:definition 'Comment: this is not the inverse of fromAgent. A message can be from someone. If we made it the inverse the person would be "to" the message'^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3808]: The original has a comment but no definition. I suggest the following:

```
skos:definition "The destination of a message or shipment" 
skos:scopeNote 'This is not the inverse of fromAgent. A message can be from someone. If we made it the inverse the person would be "to" the message."
```

The definition corresponds to that for from Agent.

Comment (uscholdm) [reply gistCore.ttl:3808]: Good idea. I tweaked your suggestion for now, but it will be up for discussion later on.

**Review by scotthenninger — APPROVED** [review]
Performed some SPARQL queries to verify that rdfs labels comments, etc. do not exist and  and skos label, alt, definition equivalents exist.

File: gistCore.ttl (hunk 4 above)
[the remark this answers is not in the document]
Comment (rjyounes) [reply gistCore.ttl:342]: This comment does not pertain to prefLabels - which I agree should be lowercase for properties, as per the convention we've agreed on - but about the other annotations where you stripped off the original prefix such as "NOTE: " or "EXAMPLE: " and are left with a scopeNote or example whose text begins with lowercase. I am recommending changing those from lower to upper.

File: gistCore.ttl [hunk 9]
```diff
@@ -1387,12 +1361,17 @@ gist:LivingThing
...
 		) ;
 	] ;
+	skos:definition "Something that is now, or at some point in time was, alive and growing."^^xsd:string ;
+	skos:example
+		"A cat, a mushroom, a tree."^^xsd:string ,
+		"Negative examples:fictional life forms such as Unicorns or Mickey Mouse."^^xsd:string
+		;
+	skos:prefLabel "Living Thing"^^xsd:string ;
+	skos:scopeNote "In the open world, you must assume that it might have since died."^^xsd:string ;
 	.
```
Comment (rjyounes) [inline gistCore.ttl:1371]: Missing space after "Negatvie example:fictional". You don't have to run this by me again for review.

File: gistCore.ttl [hunk 10]
```diff
@@ -2570,19 +2531,15 @@ gist:TimeInstant
...
-		"EXAMPLES:  A literal instant (as in 12:01.0001 January 1, 2008), or a broader but still single point in time (January 1, 2008). In the latter case, we are declaring a time instant to be an interval with no duration (or really a duration only equal to its precision)."^^xsd:string ,
-		"In gistTop all time instants are Greenwich and the local equal universal.  In gistTime we introduce local time zones"^^xsd:string ,
-		"NOTE: Our identity criteria require that something refers to each Time Instant instance."^^xsd:string
-		;
 	owl:disjointWith gist:UnitOfMeasure ;
+	skos:definition "In gistTop all time instants are Greenwich and the local equal universal.  In gistTime we introduce local time zones"^^xsd:string ;
+	skos:example "A literal instant (as in 12:01.0001 January 1, 2008), or a broader but still single point in time (January 1, 2008). In the latter case, we are declaring a time instant to be an interval with no duration (or really a duration only equal to its precision)."^^xsd:string ;
+	skos:prefLabel "Time Instant"^^xsd:string ;
+	skos:scopeNote "Our identity criteria require that something refers to each Time Instant instance."^^xsd:string ;
 	.
```
Comment (rjyounes) [inline gistCore.ttl:2540]: There is no more gistTop or gistTime. This should have been fixed with demodularization but was not caught.

File: gistCore.ttl [hunk 11]
```diff
@@ -3710,12 +3661,10 @@ gist:plannedEnd
...
 		;
-	rdfs:label "Planned End"^^xsd:string ;
-	rdfs:comment
-		"A date/time that was at least at some point in time in the future. It may be in the past now, but when we planned it, it was in the future."^^xsd:string ,
-		"NOTE:  Most frequently apples to Event(s) and Offer(s). E.g. a conference or sale offer."^^xsd:string
-		;
 	rdfs:range gist:TimeInstant ;
+	skos:definition "A date/time that was at least at some point in time in the future. It may be in the past now, but when we planned it, it was in the future."^^xsd:string ;
+	skos:prefLabel "Planned End"^^xsd:string ;
+	skos:scopeNote "Most frequently apples to Event(s) and Offer(s). E.g. a conference or sale offer."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:3667]: Should we put the conference or sale offer in a skos:example?

**Review by rjyounes — CHANGES_REQUESTED** [review]
I requested a couple of tiny tweaks. You don't need to request a re-review.

File: gistCore.ttl (hunk 9 above)
Comment (uscholdm) [reply gistCore.ttl:1371]: Done

File: gistCore.ttl (hunk 10 above)
Comment (uscholdm) [reply gistCore.ttl:2540]: @rjyounes That's outside the scope of this issue, but it was easy to do now, so I did.

File: gistCore.ttl (hunk 9 above)
Comment (uscholdm) [reply gistCore.ttl:1371]: @rjyounes 

> This comment does not pertain to prefLabels - which I agree should be lowercase for properties, as per the convention we've agreed on - but about the other annotations where you stripped off the original prefix such as "NOTE: " or "EXAMPLE: " and are left with a scopeNote or example whose text begins with lowercase. I am recommending changing those from lower to upper.

The example scopeNote on line 343 (not shown above) starts with a capital C in Coherent. I am not sure what you are suggesting.

Comment (uscholdm) [reply gistCore.ttl:1371]: @rjyounes 

> I requested a couple of tiny tweaks. You don't need to request a re-review.

I prefer that you formally approve the PR, so I requested a re-review.

File: docs/ReleaseNotes.md [hunk 12]
```diff
@@ -4,12 +4,21 @@ gist Release Notes
 Release X.x.x [ Release manager: update version number here and below in import URL]
 -----
 
-### Patch Updates
+### Minor Updates
+
+*Converted rdfs annotations to skos annotations*
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:9]: I made a couple of modifications to the release notes based on our new template. There should be a Rationale section, which I've inserted but leave to you to fill in.

**Review by rjyounes — CHANGES_REQUESTED** [review]

File: gistCore.ttl (hunk 11 above)
Comment (uscholdm) [reply gistCore.ttl:3667]: I'll leave that to the judgment of those doing the annotations refresh.  If you want to make a quick change now, feel free.

**Review by rjyounes — APPROVED** [review]

**Pull request merged**